.github/actions/aws/k8s-versions.yaml

```diff
@@ -7,10 +7,14 @@ include:
     region: eu-west-1
   - version: "1.25"
     region: ap-northeast-1
+    default: true
```

Comment:
What does default specify? Does it mean this matrix entry will run on every PR and entries that have default=false will only run when the job is being specifically scheduled?

Reply:
Exactly this